airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -215,6 +216,16 @@ def sleep_till_export_completed(self, stream_slice: Mapping[str, Any]) -> bool:
 
             sleep(self.poll_interval)
 
+    @contextlib.contextmanager
+    def _csv_response_reader(self, response: requests.Response) -> csv.DictReader:
+        tmp_file = os.path.realpath(os.path.basename(response.request.url))
+        with open(tmp_file, "wb") as data_file:
+            for chunk in response.iter_content(chunk_size=1024):
+                data_file.write(chunk)
```

Comment:
Why we cannot stream directly ?
I mean why do we store in the intermediate temporary file?